shared/ui/src/components/ListHeader/index.tsx

```diff
@@ -0,0 +1,91 @@
+/** @jsxImportSource @emotion/react */
+import styled from '@emotion/styled';
+import { Padding, FlexBox } from '../../layout';
+import { Text } from '../Text';
+import { TextPropsKey } from '../Text';
+
+
+export interface ListHeaderProps {
+    title:string,
+    description: string;
+    varient?: ListHeaderVarient;
+}
+
+type ListHeaderVarient =
+  | 'listHeader_18'
+  | 'listHeader_20'
+  | 'listHeader_24'
+  | 'listHeader_28';
+
+type ListHeaderPaddingType = {
+    [key in ListHeaderVarient]: 
+        [number,number,number,number];
+}
```

Comment:
타입을 PaddingSize로 주면 다양한 패딩 입력을 받을 수 있을 거 같아용...!

```suggestion
type ListHeaderPaddingType = {
    [key in ListHeaderVarient]: 
        PaddingSize;
}
```

Reply:
우왕!! 꿀팁!!! 수정했습니당 😍😍